inc/integrations/class-jetpack.php

```diff
@@ -54,7 +54,9 @@ public function inject_jetpack_stats_script_head_children( array $children, arra
 					'script',
 					[
 						'config' => [
-							'src' => 'https://stats.wp.com/e-' . gmdate( 'YW' ) . '.js',
+							'async' => 'async',
```

Comment:
Makes me wonder if those should be the default for our integrations somehow?

Reply:
In WP Components we had some helper methods like that. I'm not opposed to reintroducing a decent `<head>`/`<Helmet>` API.